test/types/tuple/GenericTupleWithWhereClause.chpl

```diff
@@ -0,0 +1,8 @@
+proc bar((x , y)) where isRealType(x.type) && isImagType(y.type) {
+  return x * y;
+}
+
+proc main() {
+  var n = bar((2.0, 8.0));
+  writeln(n);
+}
```

Comment:
Did you mean `(2.0, 8.0i)` and a .good file containing `16.0i`?

Reply:
Wow, good catch! Thanks!